prime-router/src/main/kotlin/fhirengine/utils/FHIRBundleHelpers.kt

```diff
@@ -339,7 +342,7 @@ internal fun getObservationExtensions(
         observationsToKeep.forEach {
             observationExtensionsToKeep.add(
                 Extension(
-                    conditionExtensionurl,
+                    conditionExtensionURL,
                     Reference(it.idBase)
                 )
```

Comment:
nit: collapse to one line

Reply:
Thanks for pointing this out. Unnecessary line expansion is a pet peeve of mine!